torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -853,7 +850,7 @@ class ContainedStmtsFinder : public IRVisitor {
     contained_.insert((Stmt*)v);
     IRVisitor::visit(v);
   }
-  void visit(const LetStmt* v) override {
+  void visit(const Block* v) override {
```

Comment:
I think the Block visitor can be removed entirely: the idea behind this IRVisitor was that it'd list all "actual" statements contained within another statement. By actual here I meant statements that might have some observable effect, I considered `Store` and `LetStmt` to be like that. It wouldn't harm if we list Blocks too, but it should not be necessary. That said, this code is in a prototype stage and will probably need to be revisited anyway, so provided all existing tests work, it should be fine.

Reply:
If LetStmt had an affect then wouldn't the var binding in block have the same effect?

Reply:
Yeah, I guess it would, you are right.